lms/djangoapps/grades/signals/handlers.py

```diff
@@ -81,7 +81,8 @@ def submissions_score_reset_handler(sender, **kwargs):  # pylint: disable=unused
     )
 
 
-def score_published_handler(block, user, new_raw_earned, new_raw_possible, only_if_higher):
+@receiver(SCORE_PUBLISHED)
+def score_published_handler(sender, block, user, raw_earned, raw_possible, only_if_higher, **kwargs):
```

Comment:
Would it be less clunky to just use `(sender, **kwargs)` and unpack `block, user, raw_earned, raw_possible, only_if_higher` from kwargs?